.github/workflows/clear_staging_bucket.yml

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: Clear staging bucket
+
+on:
+  schedule:
+    - cron: '0 0 1 * *'
+
+
+env:
+  GCP_PATH: "gs://beam-wheels-staging"
+
+
+jobs:
+  clear_staging_bucket:
+    name: Clear bucket
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Authenticate on GCP
+      uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+      with:
+        project_id: ${{ secrets.GCP_PROJECT_ID }}
+        service_account_email: ${{ secrets.GCP_SA_EMAIL }}
+        service_account_key: ${{ secrets.GCP_SA_KEY }}
+    - name: Delete folders without corresponding branch in repo
+      run: |
+        declare -a FOLDERS_TO_BE_DELETED=()
```

Comment:
I think it's much better to put it all in separate bash script and call it from there. It's easier to analyse it this way

Reply:
I agree. The bash script contained in the YAML file cannot be verified by shellcheck.

Reply:
Thanks for the suggestion. I moved it to the separate file. Looks much more nicer and it is easier do test script locally.